From 41427b54eae88e5c13471f559421b2f4cd3bd9de Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Thu, 27 Oct 2011 06:52:46 -0500
Subject: [PATCH 0521/2463] ENGR00160930-5 Add support for SSI audio on MX6
 Sabre-lite

Add support for the SSI audio through the SGTL codec
on MX6 Sabre-lite board

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 sound/soc/imx/Kconfig        |    2 +-
 sound/soc/imx/imx-sgtl5000.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/imx/Kconfig b/sound/soc/imx/Kconfig
index 61cf93e..3ea833d 100644
--- a/sound/soc/imx/Kconfig
+++ b/sound/soc/imx/Kconfig
@@ -52,7 +52,7 @@ config SND_SOC_PHYCORE_AC97
 
 config SND_SOC_IMX_SGTL5000
 	tristate "SoC Audio support for i.MX boards with sgtl5000"
-	depends on MACH_MX35_3DS || MACH_MX51_BABBAGE
+	depends on MACH_MX35_3DS || MACH_MX51_BABBAGE || MACH_MX6Q_SABRELITE
 	select SND_SOC_SGTL5000
 	select SND_MXC_SOC_MX2
 	help
diff --git a/sound/soc/imx/imx-sgtl5000.c b/sound/soc/imx/imx-sgtl5000.c
index 0ebc600..9325dc8 100644
--- a/sound/soc/imx/imx-sgtl5000.c
+++ b/sound/soc/imx/imx-sgtl5000.c
@@ -363,7 +363,7 @@ static int __init imx_sgtl5000_init(void)
 	if (ret)
 		return -ENOMEM;
 
-	if (machine_is_mx35_3ds())
+	if (machine_is_mx35_3ds() || machine_is_mx6q_sabrelite())
 		imx_sgtl5000_dai[0].codec_name = "sgtl5000.0-000a";
 	else
 		imx_sgtl5000_dai[0].codec_name = "sgtl5000.1-000a";
-- 
1.7.7.4

